internal/services/compute/image_resource.go

```diff
@@ -0,0 +1,439 @@
+package compute
+
+import (
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/profiles/2020-09-01/compute/mgmt/compute"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/commonschema"
+	"github.com/hashicorp/go-azure-helpers/resourcemanager/location"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/hashicorp/terraform-provider-azurestack/internal/az/resourceid"
+	"github.com/hashicorp/terraform-provider-azurestack/internal/az/tags"
+	"github.com/hashicorp/terraform-provider-azurestack/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurestack/internal/services/compute/parse"
+	"github.com/hashicorp/terraform-provider-azurestack/internal/tf"
+	"github.com/hashicorp/terraform-provider-azurestack/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurestack/internal/tf/suppress"
+	"github.com/hashicorp/terraform-provider-azurestack/internal/tf/timeouts"
+	"github.com/hashicorp/terraform-provider-azurestack/internal/utils"
+)
+
+func image() *pluginsdk.Resource {
+	return &pluginsdk.Resource{
+		Create: resourceImageCreateUpdate,
+		Read:   resourceImageRead,
+		Update: resourceImageCreateUpdate,
+		Delete: resourceImageDelete,
```

Comment:
we no longer put resource in the function names
```suggestion
		Create: imageCreateUpdate,
		Read:   imageRead,
		Update: imageCreateUpdate,
		Delete: imageDelete,
```

all functions will need this changed